controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
+  res
+    .status(404)
+    .send({ msg: "You tried to look for something and it wasn't found" });
+};
+
+exports.handle400s = (err, req, res, next) => {
```

Comment:
This 400 handler is not set up correctly. You will need to handle PSQL errors instead when a bad request comes up :)

Reply:
Additionally, there is no app.use set to use this middleware inside your app.js